.github/workflows/ci.yml

```
@@ -0,0 +1,104 @@
+        node-version: '20.x'
+        cache: 'npm'
+        
+    - name: Install dependencies
+      run: npm ci
+      
+    - name: Test application startup
+      run: |
+        # Start the app in background
+        timeout 10s node index.js &
+        APP_PID=$!
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)